typescript/sdk/src/middleware/account/accounts.hardhat-test.ts

```diff
@@ -78,14 +80,15 @@ describe.skip('InterchainAccounts', async () => {
       ethers.constants.AddressZero,
     );
 
-    await local['callRemote(uint32,address,uint256,bytes,bytes)'](
+    const call = { to: recipient.address, data, value: BigNumber.from(0) };
```

Comment:
test with a nonzero value too?

Reply:
don't support msg.value in the relayer so not relevant at this point.